base/system/winlogon/lang/es-ES.rc

```diff
@@ -46,6 +46,7 @@ BEGIN
     IDS_RUNNINGLOGONSCRIPTS "Ejecutando los scripts de inicio de sesión..."
     IDS_LOADINGYOURPERSONALSETTINGS "Cargando su configuración personal..."
     IDS_CLOSINGNETWORKCONNECTIONS "Cerrando las conexiones de red..."
+    IDS_REACTOSISRESTARTING "Restarting..."
     IDS_REACTOSISSHUTTINGDOWN "ReactOS se está apagando..."
```

Comment:
Since `IDS_REACTOSISSHUTTINGDOWN` now has new meaning in your PR, you need to set it the same in all translations.

Reply:
That was a very weird recommendation. It would have been better, if he wouldn't have touched IDS_REACTOSISSHUTTINGDOWN for any localization.